src/Http/Controllers/Admin/ModuleController.php

```diff
@@ -2671,7 +2690,12 @@ protected function getTransLabel($key, $replace = [])
         return twillTrans(Arr::has($this->labels, $key) ? Arr::get($this->labels, $key) : $key, $replace);
     }
 
-    public function getForm(TwillModelContract $model): Form
+    public function getForm(TwillModelContract $model): Form | Fieldsets
+    {
+        return new Form();
```

Comment:
Just an idea, but what if we `Form::fieldsets([])` this would avoid a separate class/union type?

Then we can use $formController->fieldsets() as our condition instead of the instanceof?

Reply:
This is the suggestion I was looking for. Wasn't happy about the union type but wasn't sure how to avoid it.